### DISCUSSION TITLE
Arrows on mania being smaller on osu!stable than osu!lazer

### DISCUSSION THREAD

**Discussion by Inbydev** [ppy/osu#29935] [Q&A] [answered]
(First posted on reddit and one person told me to go here) I was playing osu!lazer on Linux for a while and when I tried osu!stable on Windows, the arrows were smaller than osu!lazer in the same skin, so I can't read much:

osu!stable:
![screenshot001](https://github.com/user-attachments/assets/72850ffc-d208-46f4-9ddf-305b26de7031)

osu!lazer:
![osu_2024-09-17_19-38-59](https://github.com/user-attachments/assets/d7da23fe-3327-4bf9-8f83-06ec3a7e37e4)

It's just a bit smaller, but for me that's everything when im playing, I really can't see when its moving. Its an option or something about osu!lazer or osu!stable? because I try other skins and those work fine I think. Also, the skin is exported from lazer, but I tried downloading the same skin from the video I saw the skin the first time and nothing. I really don't know why this happens. I also use a 60Hz monitor. I have an 8000Hz polling rate keyboard, but I only use 1000Hz on osu!lazer on Linux because I had problems before. For that reason, I changed the polling rate to 1000Hz on Windows too, maybe that fix it, but nothing too.

Skin: [R Skin v3.0 (Narrow Bars) [Retsukiya] [Inby edit]](https://www.mediafire.com/file/cab977bjf6cibul/R_Skin_v3.0_%2528Narrow_Bars%2529_%255BR-skin%255D_%2528Retsukiya%2529.osk/file)

Video with the skin I saw with the timestamp: https://www.youtube.com/watch?v=bpZI4LXJlMs&t=190s

**Comment by RubiksImplosion** [comment, marked as answer]
probably #24664

**Reply by Inbydev** [reply]
maybe :o let me try it, thanks you!

**Reply by Inbydev** [reply]
thanks you so much! it is that!! thx u really much!!!
